### ISSUE TITLE
Save CLASS transfer functions when calculating matter power

### ISSUE THREAD

**Issue by dkirkby** [LSSTDESC/CCL#114]
This could either be triggered by an option to `ccl_cosmology_compute_power_class()` or we could always calculate and save the transfer functions (since they are presumably always being calculated anyway) to simplify the API.  Any opinions?

**Comment by c-d-leonard** [comment]
I believe we still do not save the transfer functions. Is this something we still want? I can see it being useful, especially for modified gravity applications. @damonge @dkirkby?

**Comment by damonge** [comment]
Yes, this is eventually gonna bite us in the ass. We have a relatively nice formalism to store these things (the generalized Pks), but we're not using it for it.

**Comment by dkirkby** [comment]
Yes, I think we still want this.  Is it easier to implement now?

**Comment by damonge** [comment]
I think so, and it could be done at the python level (the same way it's done for the power spectrum now)
